code/modules/research/xenobiology/xenobiology.dm

```diff
@@ -589,16 +589,16 @@
 
 /obj/item/areaeditor/blueprints/slime
 	name = "cerulean prints"
-	desc = "A one use set of blueprints made of jelly like organic material. Renaming an area to 'Xenobiology Lab' will extend the reach of the management console."
+	desc = "A one use set of blueprints made of jelly like organic material. Extends the reach of the management console."
 	color = "#2956B2"
 
 /obj/item/areaeditor/blueprints/slime/edit_area()
-	. = ..()
+	..()
 	var/area/A = get_area(src)
-	if(.)
-		for(var/turf/T in A)
-			T.color = "#2956B2"
-		qdel(src)
+	for(var/turf/T in A)
+		T.color = "#2956B2"
```

Comment:
We might want a CHECK_TICK in this loop, areas can get pretty big.

Reply:
You sure? It's just changing the color.

Reply:
Not worth the CPU cycles it saves.